backpropagation/network/neural_network.py

```diff
@@ -41,16 +41,34 @@ def __init__(
         self.layers_count = len(neurons_count_per_layer)
 
         self.weights = [
-            np.random.normal(size=(
+            self._create_weight_matrix(size=(
                 neurons_count_per_layer[layer + 1],
-                neurons_count_per_layer[layer] + 1
+                neurons_count_per_layer[layer]
             ))
             for layer in range(self.layers_count - 1)
         ]
-
         self.activation_function = activation_function
         self.cost_function = cost_function
 
+    def _create_weight_matrix(self, size: Tuple[int, int]) -> np.array:
+        """Creates a weight matrix between layers of sizes specified by ``size``.
+
+            It creates the biases weights and stacks it horizontally in front of the
```

Comment:
If `biases = np.random.normal(size=(rows, 1))` it seems like it's column vector, which cannot not be added horizontally.

Reply:
In numpy horizontal stacking refers to situation where a, b are stacked like this: `a|b`. I wanted to remain consistent with the code.